.github/workflows/triage.yml

```diff
@@ -66,52 +69,52 @@ jobs:
                 }, {
                     "label": "go",
                     "path": "Formula/.+",
-                    "content": "depends_on \"go(?:@[0-9.]+)?\""
+                    "content": "depends_on \"go(@[0-9.]+)?\""
                 }, {
                     "label": "haskell",
                     "path": "Formula/.+",
-                    "content": "depends_on \"(?:ghc@?|haskell-stack).+"
+                    "content": "depends_on \"(ghc|haskell-stack)(@[0-9.]+)?\""
                 }, {
                     "label": "java",
                     "path": "Formula/.+",
-                    "content": "depends_on \"?(?:java|openjdk@?).+"
+                    "content": "depends_on \"openjdk(@[0-9.]+)?\""
                 }, {
                     "label": "linux-only",
                     "path": "Formula/.+",
                     "content": "depends_on :linux"
                 }, {
                     "label": "lua",
                     "path": "Formula/.+",
-                    "content": "depends_on \"lua(?:@[0-9.]+|jit)?\""
+                    "content": "depends_on \"lua(jit)?(@[0-9.]+)?\""
```

Comment:
Can you also include `luajit-openresty` here?

Reply:
Oh yeah thanks for reminding me